be/src/common/config.h

```diff
@@ -414,8 +414,12 @@ CONF_mInt32(streaming_load_rpc_max_alive_time_sec, "1200");
 // the timeout of a rpc to open the tablet writer in remote BE.
 // short operation time, can set a short timeout
 CONF_Int32(tablet_writer_open_rpc_timeout_sec, "60");
+// the timeout of a rpc to open the partition in remote BE.
+// short operation time, can set a short timeout
+CONF_Int32(partition_open_rpc_timeout_sec, "60");
```

Comment:
please add a flag in config.h to enable/disable this feature for compatibility reasons when the user is doing upgrade.